CHANGELOG.md

```diff
@@ -22,6 +27,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Use `0.1.1` tag for `k8s-api-heahtz` image.
 - Use `0.1.0` tag for `k8s-setup-network-env` image.
 - Use `0.1.0` tag for `aws-attach-etcd-dep` image.
+- Use `0.1.0` tag for `k8s-setup-network-env` image.
+- Use `0.1.1` tag for `k8s-api-heahtz` image.
+
+### Fixed
+
+- Fix failing go template rendering of KMS encryption content.
```

Comment:
This here does not look right. It is already mentioned above.

Reply:
Thanks, fixed.